public/modules/custom/grants_budget_components/js/budget_total.js

```diff
@@ -0,0 +1,77 @@
+// eslint-disable-next-line no-unused-vars
+((Drupal, drupalSettings) => {
+  Drupal.behaviors.grants_budget_total_fieldAccessData = {
+    attach: function attach() {
+
+      function calculateCostTotal() {
+        var sumCost = 0;
+
+        // Loop through the input fields and add their values to the sum
+
+        drupalSettings.costTotalFields.forEach(costTotalField => {
+
+          let costTotalFieldName = 'edit-suunnitellut-menot-' + costTotalField.name
```

Comment:
Tässä oletetaan, että kenttä olisi aina suunnitellut menot - Jos tän pitäis toimia kaikilla lomakkeilla (?) saatta olla jonkinlainen ongelma, kun noi voi olla eri nimillä ja niitä voi olla useampi - Esim tulevassa lomakkeessa https://github.com/City-of-Helsinki/hel-fi-drupal-grants/pull/593 on noita meno kenttiä yhteensä 5 kappaletta.

Reply:
Onko ehdotuksia tähän? :D

Reply:
Varmaan pitäis keskustella vähän isomman poppoon kanssa, että mihin kaikkeen tätä käytetään ja millä käyttötarkotuksilla. Jos siihen ilmestyykin joku ihan överin kompleksi vaatimus, niin varmaa melkein helpompi lisätä komponenttiin joku konfiguraatiokenttä, johon vois sitte vaikka manuaalisesti hakata halutut selectorit.